addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -643,6 +643,7 @@ export class OdooEditor extends EventTarget {
         this.addDomListener(this.document, 'mouseup', this._onDocumentMouseup);
         this.addDomListener(this.document, 'click', this._onDocumentClick);
         this.addDomListener(this.document, 'scroll', this._onScroll, true);
+        this.addDomListener(this.document, 'dblclick', this._onDoubleClick);
```

Comment:
I didn't see this in the task description. So now I tested it and in 16.0 I couldn't reproduce the bug. What I _could_ reproduce was that two cells were selected when _triple_ clicking. So it seems the problem would have to do with triple-click correction, and shouldn't be fixed by forcing a cell selection on double click.

Reply:
Yes, it was not mentioned in task pad but shra-odoo told that on doubleClick should select single cell.

Yes, you are right about triple click that it select two cells on triple click.We can solve that using `correctTripleClick: true` by which selection will stay in single cell only but it also creates some problems like we can't select row and we try to do that thewn it will select full table. I am confused about how to tackle it as i don't have idea about how can i detect tripleClick event in `_handleSelectionInTable` and we don't  have `ev` also.What kind of conditon i can put in `correctTripleClick` so that it will work only for triple click.

Reply:
Yes, i'm ok with it.